sswlib/src/main/java/components/Mech.java

```diff
@@ -3165,12 +3165,12 @@ public double GetChassisCost() {
 
     public double GetTotalCost() {
         // final cost calculations
-        return ( GetEquipCost() + GetChassisCost() ) * GetCostMult();
+        return ((GetEquipCost() + GetChassisCost()) * GetCostMult()) + GetAmmoCosts();
     }
 
     public double GetDryCost() {
         // returns the total cost of the mech without ammunition
-        return ( GetEquipCost() + GetChassisCost() ) * GetCostMult();
+        return (GetEquipCost() + GetChassisCost()) * GetCostMult();
     }

```

Comment:
Like with the vehicles, Next time this area is touched, `GetTotalCost` should reference `GetDryCost` to minimize code duplication.

Reply:
I can update these when I add VTOL.